File: 1742-widest-vertical-area-between-two-points-containing-no-points/1742-widest-vertical-area-between-two-points-containing-no-points.java
class Solution {
    public int maxWidthOfVerticalArea(int[][] points) {
        int l = points.length;
        int[] x = new int[l];
        for (int i = 0; i < l; i++) {
            x[i] = points[i][0];
        }
        Arrays.sort(x);
        int max = 0;
        for (int i = 1; i < l; i++) {
            int d = x[i] - x[i - 1];
            if (d > max) {
                max = d;
            }
        }
        return max;
    }
}

File: 1742-widest-vertical-area-between-two-points-containing-no-points/1742-widest-vertical-area-between-two-points-containing-no-points.py
class Solution:
    def maxWidthOfVerticalArea(self, points: List[List[int]]) -> int:
        l = len(points)
        xcoords = [0] * l
        for i in range(l):
            xcoords[i] = points[i][0]
        xcoords.sort()
        mx = 0
        for i in range(1, len(xcoords)):
            d = xcoords[i] - xcoords[i - 1]
            if d > mx:
                mx = d
        return mx